src/classes/PobControllerExt.cls

```diff
@@ -1,4 +1,9 @@
-public with sharing class PobControllerExt {
+public class PobControllerExt {
+
+
+    public void empty() {
+        // do nothing. let's add stuff later.
+    }
```

Comment:
Class `PobControllerExt` does not enforce sharing correctly. Please change to `with sharing`.